modules/stereo_corresp/include/opencv2/calib3d/calib3d.hpp

```diff
@@ -0,0 +1,48 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
```

Comment:
please, do not copy the headers. create new one "opencv2/stereo.hpp". there is no need in nested "opencv2/stereo/stereo.hpp".